plugin/src/main/scala/se/nullable/sbtix/NixWriter.scala

```diff
@@ -68,7 +68,7 @@ case class NixRepoCollection(repos: Seq[NixRepo])  extends NixBuilder {
 
   def toNixValue: String =
       s"""{
-          |  ${indent(repos.map(_.toNix).mkString("\r\n"))}
+          |  ${indent(repos.distinct.map(_.toNix).mkString("\r\n"))}
```

Comment:
Perhaps it would make more sense to have repos be a Set? Admittedly, the poor use of Seq was my fault, but if we're deduping it anyway...